packages/neo-one-monitor/src/MonitorBase.js

```diff
@@ -777,6 +785,38 @@ export default class MonitorBase implements Span {
     }
   }
 
+  report(report: Report): void {
+    report.logs.map(log => this._logger.log(log));
+
+    for (const counterMetric of report.metrics.counters) {
+      const counter = this.getCounter({
+        name: counterMetric.metric.name,
+        help: counterMetric.metric.help,
+        labelNames: counterMetric.metric.labelNames,
+        metricLabels: counterMetric.metric.metricLabels,
+        receivedFromBrowser: true,
+      });
+
+      counterMetric.values.forEach(value => {
+        counter.inc(value.countOrLabels, value.count);
```

Comment:
This isn't going to work anymore with the latest changes (might need to rebase). `getCounter` returns a wrapped metric that adds the current monitor's labels.

Reply:
Probably want to disable returning the wrapper when `receivedFromBrowser` is true.